make/dev.mk

```
@@ -34,7 +34,10 @@ deploy-e2e-to-dev-namespaces:
 .PHONY: deploy-e2e-to-dev-namespaces-two-members
 deploy-e2e-to-dev-namespaces-two-members:
 	$(MAKE) deploy-e2e MEMBER_NS=${DEV_MEMBER_NS} MEMBER_NS_2=${DEV_MEMBER_NS_2} HOST_NS=${DEV_HOST_NS} REGISTRATION_SERVICE_NS=${DEV_REGISTRATION_SERVICE_NS} ENVIRONMENT=${DEV_ENVIRONMENT} E2E_TEST_EXECUTION=false IS_OSD=${IS_OSD}
-	if [[ ${DEV_SSO} == true ]]; then $(MAKE) run-cicd-script SCRIPT_PATH=scripts/ci/setup-dev-sso.sh  SCRIPT_PARAMS="--sso-ns $(DEV_SSO_NS)"; fi
+	if [[ ${DEV_SSO} == true ]]; then \
+		$(MAKE) download-assets ASSETS_FOLDER=scripts/ci/dev-sso; \
+		$(MAKE) run-cicd-script SCRIPT_PATH=scripts/ci/setup-dev-sso.sh  SCRIPT_PARAMS="--sso-ns $(DEV_SSO_NS)"; \
+	fi
```

if `download-assets` fails, we should not `run-cicd-script`

```suggestion
	if [[ ${DEV_SSO} == true ]]; then \
		$(MAKE) download-assets ASSETS_FOLDER=scripts/ci/dev-sso && \
		$(MAKE) run-cicd-script SCRIPT_PATH=scripts/ci/setup-dev-sso.sh  SCRIPT_PARAMS="--sso-ns $(DEV_SSO_NS)"; \
	fi
```

same applies at L52 and L59

---

good catch!